drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -273,7 +272,11 @@ matterManufacturer:
     vendorId: 0x1168
     productId: 0x03f8
     deviceProfileName: light-color-level-1800K-6500K
-
+  - id: "4456/1024"
+    deviceLabel: Smart Plug
```

Comment:
Just confirming - they want this to just show up as "Smart Plug", no brand name?

Reply:
Yes - that is the way they submitted it. We have recommended formats that I provided to the WWST team to help promote consistency.